src/arkit.cs

```diff
@@ -651,6 +766,20 @@ interface ARSession {
 		[MarshalDirective (NativePrefix = "xamarin_simd__", Library = "__Internal")]
 		[Export ("createReferenceObjectWithTransform:center:extent:completionHandler:")]
 		void CreateReferenceObject (Matrix4 transform, Vector3 center, Vector3 extent, Action<ARReferenceObject, NSError> completionHandler);
+
+		[iOS (13,0)]
+		[Export ("raycast:")]
+		ARRaycastResult[] Raycast (ARRaycastQuery query);
+
+		[iOS (13,0)]
+		[Async]
+		[Export ("trackedRaycast:updateHandler:")]
+		[return: NullAllowed]
+		ARTrackedRaycast TrackedRaycast (ARRaycastQuery query, Action<NSArray<ARRaycastResult>> updateHandler);
```

Comment:
Could this be an array [] ?

Reply:
Yes, it can, and I've changed it accordingly.